test/statetest/statetest_loader.cpp

```diff
@@ -140,6 +140,20 @@ inline uint64_t calculate_current_base_fee_eip1559(
     return base_fee;
 }
 
+namespace
+{
+bytes32 load_bytes32(const json::json::const_iterator& json_it)
+{
+    // Special case to handle "0". Required by exec-spec-tests.
+    // TODO: Get rid of it.
+    if (json_it->is_string() &&
+        (json_it->get<std::string>() == "0" || json_it->get<std::string>() == "0x0"))
```

Comment:
```suggestion
        (*json_it == "0" || *json_it == "0x0"))
```
I believe you can shorten this.